gallery-code/codeworld/PEwIFcw4EXZtdP7JdytExKA.hs

```diff
@@ -0,0 +1,57 @@
+main = collaborationOf(2, initial, step, event, picture)

```

Comment:
Hmm, ok. I will bundle these files with the `Vagrantfile` script that builds the image then instead.